swarm/src/handler/pending.rs

```diff
@@ -56,8 +56,8 @@ impl ConnectionHandler for PendingConnectionHandler {
         void::unreachable(v)
     }
 
-    fn connection_keep_alive(&self) -> KeepAlive {
-        KeepAlive::No
+    fn connection_keep_alive(&self) -> bool {
+        false
     }
```

Comment:
Given the new default implementation, we can remove this everywhere where we statically return `false`.

Reply:
removed 3d10eb4